### ISSUE TITLE
Using F to open a link in a new tab does so twice

### ISSUE THREAD

**Issue by clompi104** [philc/vimium#4032]
### Issue:

- The problem: Using F (shift+f) to open any link in a new tab opens two new tabs each loaded with the specified link.
- Expected behavior: Doing the described action should result in just one new tab opened with the specified link loaded.

### Technical specifications:

- OS: Windows 10 64-bit
- Browser: Firefox 97.0.2 (64-bit) (privacy-oriented hardened about:config)
- Vimium: Vimium-FF 1.67.1 from the Firefox addons store

**Comment by Vovchikan** [comment]
Have the dame issue

- Os: Debian GNU/Linux 10 (buster)
- Browser: Firefox 91.6.1esr (64-bit)
- Vimium: 1.67.1 from Fifefox store

**Comment by gdh1995** [comment]
Sounds like #3000 can fix this bug.

**Comment by frankfarm** [comment]
Same.
- OS: macOS 12.3.1 Monterey
- Browser: 91.8.0esr (64-bit)
- Vimium-FF: 1.67.1 from addons.mozilla.org

**Comment by l1yefeng** [comment]
Same.

- OS: Debian GNU/Linux 11 (bullseye)
- Firefox: Mozilla Firefox 91.9.0esr
- Vimium-FF: v1.67.1

**Comment by npn225** [comment]
Same.

- OS: EndeavourOS Linux x86_64
- Firefox: Mozilla Firefox 100.0
- Window Manager: i3
- Vimium-FF: v1.67.1

**Comment by gdh1995** [comment]
@npn225 Have you enabled `privacy.resistFingerprinting` on Firefox's `about:config` page?

**Comment by npn225** [comment]
@gdh1995  Yes, I have enabled `privacy.resistFingerprinting` on Firefox's `about:config` page. 

I know you have created a fix for this specific issue at #4000 (thank you very much for that) but I didn't want to go through all the trouble of cloning the git repository for this extension, pulling your fixes/changes, and then finally disabling the original add-on I got from the Firefox Web-Store so that I can import the git clone version of it into my Firefox browser.  

Too much work when I can either just disable `privacy.resistFingerprinting` or just not use the "F" option for Vimium-FF.
